frontend/pages/admin/TeamManagementPage/TeamDetailsWrapper/TeamDetailsWrapper.tsx

```diff
@@ -519,6 +528,7 @@ const TeamDetailsWrapper = ({
           onCancel={toggleDeleteTeamModal}
           onSubmit={onDeleteSubmit}
           name={currentTeam.name}
+          teamIsRemoving={teamIsRemoving}
```

Comment:
just a pass-by comment, feel free to ignore: what do you think of `isLoading` (or similar) as the name for these kind of props instead of the more specific `teamIsRemoving`, `teamIsEditing`, `isEditingUser`, etc?

Reply:
Thanks for the feedback! The same occurred to me. I went this route because there are multiple different loading states in the parent component, so it's necessary to indicate what type of loading state it is. However, that doesn't mean the `DeleteTeamModal` couldn't have the more generic name, which is clearer. So, for example, this could change to `isLoading={teamIsRemoving}` or something similar. I'll push up another commit to clean this up. 👍